### PR TITLE
AS-1000: Modernize README

### PR DESCRIPTION
**Pull request by MatthewBemis**
- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check that the **Product Owner** has signed off on any user-facing changes
- [ ] **Submitter**: Make sure Swagger is updated if API changes
  - [ ] **...and Orchestration's Swagger too!**
- [ ] **Submitter**: If updating admin endpoints, also update [firecloud-admin-cli](https://github.com/broadinstitute/firecloud-admin-cli)
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Database checks:
  * If PR includes new or changed db queries, include the explain plans in the description
  * Make sure liquibase is updated if appropriate
  * If doing a migration, take a backup of the
  [dev](https://console.cloud.google.com/sql/instances/terraform-qfarbdq3lrexxck5htofjs5z6m/backups?project=broad-dsde-dev&organizationId=548622027621)
  and
  [alpha](https://console.cloud.google.com/sql/instances/terraform-r4caezzc35c4tb7pgdhwkmme4y/backups?project=broad-dsde-alpha&organizationId=548622027621)
  DBs in Google Cloud Console
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] Tell your tech lead (TL) that the PR exists if they want to look at it
- [ ] Anoint a lead reviewer (LR). **Assign PR to LR**
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Assign to submitter** to finalize
- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Squash commits and merge to develop
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Verify swagger UI on dev environment still works after deployment
- [ ] **Submitter**: Inform other teams of any API changes via Slack and/or email
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +23,51 @@ sbt antlr4:antlr4Generate # Generates source code for IntellIJ IDEA
 sbt clean compile test
```
Comment (sagehen03) [inline README.md]: When I run this, I get these error messages:
```
[error] Database host name not set. Please see run instructions in README.md for providing a valid database hostname
[error] Database host name not set. Please see run instructions in README.md for providing a valid database hostname
[error] Database host name not set. Please see run instructions in README.md for providing a valid database hostname
[error] Database host name not set. Please see run instructions in README.md for providing a valid database hostname
[error] Database host name not set. Please see run instructions in README.md for providing a valid database hostname
[error] Database host name not set. Please see run instructions in README.md for providing a valid database hostname
```

File: README.md [hunk 2]
```diff
@@ -21,7 +23,51 @@ sbt antlr4:antlr4Generate # Generates source code for IntellIJ IDEA
...
-See the wiki for detailed documentation.
+## Running Locally
+
+### Requirements:
+
+* [Docker Desktop](https://www.docker.com/products/docker-desktop) (4GB+, 8GB recommended)
+* Broad internal internet connection (or VPN, non-split recommended)
+* Render the local configuration files. From the root of the [firecloud-develop](https://github.com/broadinstitute/firecloud-develop) repo, run:
+```sh
+./firecloud-setup.sh
```
Comment (sagehen03) [inline README.md]: I tried this step to resolve the database host name not set errors from above, but I don't see such a file in the root of the firecloud-develop repo.

File: README.md (hunk 1 above)
Comment (MatthewBemis) [reply README.md]: Ah, I should remove this line from `Getting Started`

Refer to the section titled `Unit Testing with MySQL in Docker`

File: README.md (hunk 2 above)
Comment (MatthewBemis) [reply README.md]: I've fixed the path for this file in the latest commit

File: README.md [hunk 3]
```diff
@@ -50,6 +43,58 @@ And when you're done, spin down mysql (it is also fine to leave it running for y
...
 
+## Running Locally
+
+### Requirements:
+
+* [Docker Desktop](https://www.docker.com/products/docker-desktop) (4GB+, 8GB recommended)
+* Broad internal internet connection (or VPN, non-split recommended)
+* Render the local configuration files. From the root of the [firecloud-develop](https://github.com/broadinstitute/firecloud-develop) repo, run:
+```sh
+sh run-context/local/scripts/firecloud-setup.sh
```
Comment (sagehen03) [inline README.md:55]: It looks like this script requires a ruby interpreter installed at `/usr/bin/ruby` easy enough to work around but might be worth calling out earlier in terms of prerequisites.

File: README.md [hunk 4]
```diff
@@ -50,6 +43,58 @@ And when you're done, spin down mysql (it is also fine to leave it running for y
...
+```sh
+127.0.0.1	local.broadinstitute.org
+```
+
+### Running:
+
+After satisfying the above requirements, execute the following command from the root of the Rawls repo:
+
+```sh
+./config/docker-rsync-local-rawls.sh
```
Comment (sagehen03) [inline README.md:69]: I get an error at this point about a missing config file..
```
com.typesafe.config.ConfigException$IO: /app/config/built-config.conf: java.io.FileNotFoundException: /app/config/built-config.conf (No such file or directory)
```

Comment (sagehen03) [reply README.md:69]: Looks like I need to run as root and that solved the issue.

File: README.md [hunk 5]
```diff
@@ -65,6 +44,58 @@ And when you're done, spin down mysql (it is also fine to leave it running for y
...
+
+### Requirements:
+
+* [Docker Desktop](https://www.docker.com/products/docker-desktop) (4GB+, 8GB recommended)
+* Broad internal internet connection (or VPN, non-split recommended)
+* Render the local configuration files. From the root of the [firecloud-develop](https://github.com/broadinstitute/firecloud-develop) repo, run:
+```sh
+sh run-context/local/scripts/firecloud-setup.sh
+```
+Note: this script will offer to set up configuration for several other services as well. You can skip those if you only want to set up configuration for Rawls. If this is your first time running Rawls or rendering congiguration files, you will want to run through the "Setup vault" step.
```
Comment (jordansafer) [inline README.md]: Nit: congiguration -> configuration in the second sentence :)

Comment (MatthewBemis) [reply README.md]: how embarrassing. just fixed!

**Review by jordansafer — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (MatthewBemis) [reply README.md:55]: added a note about this below

**Pull request merged**